File: backend/ConAlambreApi/Models/PerfilProfesional.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ConAlambreApi.Models
{
    public class PerfilProfesional
    {
        [Key]
        public int Id { get; set; }
        public string Titulo { get; set; }
        public string Experiencia { get; set; }
        public string Descripcion { get; set; }
        public int UsuarioId { get; set; }
        
        [ForeignKey("UsuarioId")]
        public Usuario Usuario { get; set; }
        
        public ICollection<Categoria> Categorias { get; set; } = new List<Categoria>();
    }
}

File: frontend/src/types/perfilProfesional.ts
export interface CreatePerfilProfesionalRequest {
  usuarioId: number;
  experiencia: string;
  descripcion: string;
  categoriaIds: number[];
}

export interface PerfilProfesionalResponse {
  id: number;
  titulo: string;
  experiencia: string;
  descripcion: string;
  categorias: { id: number; nombre: string; descripcion: string }[];
  usuarioId: number;
}
